test/mixer_test.py

```diff
@@ -169,88 +162,65 @@ def get_bytes(snd):
             uwave_path = unicode_(wave_path)
             bwave_path = uwave_path.encode(sys.getfilesystemencoding())
             snd = mixer.Sound(file=wave_path)
-            self.assert_(snd.get_length() > 0.5)
+            self.assertTrue(snd.get_length() > 0.5)
             snd_bytes = get_bytes(snd)
-            self.assert_(len(snd_bytes) > 1000)
-            self.assert_(get_bytes(mixer.Sound(wave_path)) == snd_bytes)
-            self.assert_(get_bytes(mixer.Sound(file=uwave_path)) == snd_bytes)
-            self.assert_(get_bytes(mixer.Sound(uwave_path)) == snd_bytes)
+            self.assertTrue(len(snd_bytes) > 1000)
+            self.assertEqual(get_bytes(mixer.Sound(wave_path)), snd_bytes)
+            self.assertEqual(get_bytes(mixer.Sound(file=uwave_path)), snd_bytes)
+            self.assertEqual(get_bytes(mixer.Sound(uwave_path)), snd_bytes)
             arg_emsg = 'Sound takes either 1 positional or 1 keyword argument'
-            try:
+
+            with self.assertRaises(TypeError) as cm:
                 mixer.Sound()
-            except TypeError:
-                self.assertEqual(str(geterror()), arg_emsg)
-            else:
-                self.fail("no exception")
-            try:
+                self.assertEqual(cm.exception, arg_emsg)
```

Comment:
oups, I just realized, there is a mistake here (we never enter the `assertEqual`). It should be one indentation lower, and then it needs to be spelled like : 
```python
self.assertEqual(cm.exception.message, arg_emsg)
```
 (or `str(cm.exception)` ...)